internal/command/a.go

```diff
@@ -113,12 +113,18 @@ type RHash interface {
 	Values(key string) ([]core.Value, error)
 }
 
+// RSet is a set repository.
+type RSet interface {
```

Comment:
It's better not to rush things with the `command` package. Let's deal with `rset` first.

Reply:
Shouldn't I implement the `internal/command` part?